src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -5237,6 +5271,7 @@ namespace Google.Example.Library.V1.Tests
 
 using gax = Google.Api.Gax;
 using gaxgrpc = Google.Api.Gax.Grpc;
+using gaxres = Google.Api.Gax.ResourceNames;
```

Comment:
Is this hard-coded somewhere? I'm just trying to work out what happens for the Bigtable scenario, where we might have Google.Api.Gax.ResourceNames.ProjectName and also Google.Cloud.Bigtable.CommonResources.V1.TableName. (I'm not sure what we're going to do about versioning for common names, admittedly. That could be fun.)

Reply:
Yes, this specific namespace is hardcoded to the `gaxres` alias.
In the example you give, `Google.Cloud.Bigtable.CommonResources.V1` will be imported using a standard (non-aliasing) `using` directive.
We don't yet have any use-cases where aliases of arbitrary namespaces are required. We should probably discuss the best way to handle these, but I don't think that needs to be part of this PR.

Reply:
Okay, I'm fine with that, so long as there's a plan :)